lib/gds-sso/config.rb

```diff
@@ -11,13 +12,15 @@ module Config
 
       # OAuth ID
       mattr_accessor :oauth_id
+      @@oauth_id = ENV.fetch("OAUTH_ID", "test-oauth-id")
```

Comment:
I feel like the environment variables are rather generically named - which was a pre-existing problem but feels a bit worse now you couldn't find usage in the codebase with a grep. It'd be more conventional for them to be prefixed with `GDS_SSO_` but can understand that'd be a bit of puppet pain

Reply:
Yes, I agree the naming is short-sighted and could be more precise. I do think it's out-of-scope for the purpose of this change, since we don't currently have a clear conflict with these variables (until...accounts).

The naming doesn't seem to affect their discoverability, but they will become more hidden as a consequence of this change. I've added a couple of commits to refresh the README to help, which I also forgot to do earlier.

Reply:
Sure no worries and agree it was out of scope for here. While it's fresh in minds I think I'm going to have a go at switching them to have a GDS_SSO prefix - unless you have reason to object?

Reply:
Sure, that's fine with me. I guess it's not too hard to do, although you'll need to do a fair few app deployments!